third_party/identify_stream_usage/identify_stream_usage.cpp

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
We can directly use the code by keeping license, right? We only use it from third party and in testing purpose.

Reply:
yes, maybe we should mention it in LICENSE.md, but the licenses are compatible